README.md

```diff
@@ -203,77 +243,46 @@ If you are looking for the codebase of the current production version of GATK, p
-       * unset or any other value         : run non-cloud unit and integration tests, this is the default
-       * `cloud`, `unit`, `integration`   : run only the cloud, unit, or integration tests
-       * `all`                            : run the entire test suite
+* Running the CNV case and PoN creation Workflows with premade Queue scripts:
+    * [Broad Internal instructions](http://gatkforums.broadinstitute.org/gatk/discussion/6786/howto-run-gatk-cnv-using-premade-queue-scripts-broad-internal)
 
-         
-    * Note that `git lfs` must be installed and set up as described in the "Requirements" section above
-      in order for all tests to pass.
-    * Cloud tests require being logged into `gcloud` and authenticated with a project that has access
-      to the test data.  They also require setting several certain environment variables.
-      * `HELLBENDER_TEST_PROJECT` : your google cloud project 
-      * `HELLBENDER_TEST_APIKEY` : your google cloud API key
-      * `HELLBENDER_TEST_STAGING` : a gs:// path to a writable location
-      * `HELLBENDER_TEST_INPUTS` : path to cloud test data, ex: gs://hellbender/test/resources/
       
-    * setting the environment variable `TEST_VERBOSITY=minimal` will produce much less output from the test suite 
-
-* To run a subset of tests, use gradle's test filtering (see [gradle doc](https://docs.gradle.org/current/userguide/java_plugin.html)), e.g.,
-    * `./gradlew test --tests *SomeSpecificTestClass`
-    * `./gradlew test --tests all.in.specific.package*`
-    * `./gradlew test --tests *SomeTest.someSpecificTestMethod`
-
-* To run tests and compute coverage reports, run **`./gradlew jacocoTestReport`**. The report is then in `build/reports/jacoco/test/html/index.html`.
-  (IntelliJ has a good coverage tool that is preferable for development).
-
-* We use [Travis-CI](https://travis-ci.org/broadinstitute/gatk) as our continuous integration provider.
-
-    * Before merging any branch make sure that all required tests pass on travis.
-    * Every travis build will upload the test results to our gatk google bucket.
-      A link to the uploaded report will appear at the very bottom of the travis log.
-      Look for the line that says `See the test report at`.
-      If TestNG itself crashes there will be no report generated.
-
-* We use [Broad Jenkins](https://gatk-jenkins.broadinstitute.org/view/Performance/) for our long-running tests and performance tests.
-    * To add a performance test (requires Broad-ID), you need to make a "new item" in Jenkins and make it a "copy" instead of a blank project. You need to base it on either the "-spark-" jobs or the other kind of jobs and alter the commandline. 
-
-* To output stack traces for `UserException` set the environment variable `GATK_STACKTRACE_ON_USER_EXCEPTION=true`
+## <a name="developers">For GATK Developers</a>
 
-## General guidelines for GATK4 developers
+#### <a name="dev_guidelines">General guidelines for GATK4 developers</a>
```

Comment:
We had a discussion about this recently, and decided to add a table of contents rather than break this up into separate documents, because with everything on one page we are able to search easily in the README, which is super-useful to us.